fusilli-plugin/.gitignore

```diff
@@ -0,0 +1,8 @@
+# CMake build
```

Comment:
The Apache OSS header is apparently actually optional https://softwareengineering.stackexchange.com/a/357796. Seemed a little silly on a 1 line configuration file 🤷‍♂️ . I don't feel strongly if it's preferable to add it.